syncer/syncer.go

```diff
@@ -700,63 +694,63 @@ func (s *Syncer) Process(ctx context.Context, pr chan pb.ProcessResult) {
 	}
 }
 
-func (s *Syncer) getTable(tctx *tcontext.Context, origSchema, origTable, renamedSchema, renamedTable string) (*model.TableInfo, error) {
-	ti, err := s.schemaTracker.GetTable(origSchema, origTable)
+func (s *Syncer) getTable(tctx *tcontext.Context, origTable, targetTable *filter.Table) (*model.TableInfo, error) {
```

Comment:
perfer change this to `getTableInfo`

Reply:
also need change `schemaTracker.GetTable`  to `schemaTracker.GetTableInfo` maybe in next PR 😅